### PR TITLE
pythonPackages.python-tado: init at 0.9.0

### PR DESCRIPTION
**Pull request by JamieMagee**
###### Motivation for this change

Support `tado` integration in Home Assistant

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/python-tado/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    rev = "516aa399a9ca158d3f4981605432062df502d1bd";
+    sha256 = "0cr5sxdvjgdrrlhl32rs8pwyay8liyi6prm37y66dn00b41cb5l3";
+  };
+
+  propagatedBuildInputs = [ requests ];
+  checkInputs = [ pytestCheckHook ];
+
+  meta = with lib; {
+    description =
+      "PyTado from chrism0dwk, modfied by w.malgadey, diplix, michaelarnauts, LenhartStephan, splifter, syssi, andersonshatch, Yippy, p0thi";
```
Comment (prusnak) [inline pkgs/development/python-modules/python-tado/default.nix:21]: The description on PyPI is useless. This one from GitHub is better:

```suggestion
      "Python binding for Tado web API. Pythonize your central heating!";
```

**Review by prusnak — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/python-tado/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+buildPythonPackage rec {
+  pname = "python-tado";
+  version = "0.9.0";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "wmalgadey";
+    repo = "PyTado";
+    rev = "516aa399a9ca158d3f4981605432062df502d1bd";
```
Comment (prusnak) [inline pkgs/development/python-modules/python-tado/default.nix:12]:
```suggestion
    rev = version;
```

**Review by prusnak — CHANGES_REQUESTED** [review]
Missed one

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107956` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.python-tado</li>
    <li>python38Packages.python-tado</li>
    <li>python39Packages.python-tado</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107956` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.python-tado</li>
    <li>python38Packages.python-tado</li>
    <li>python39Packages.python-tado</li>
  </ul>
</details>

File: pkgs/development/python-modules/python-tado/default.nix (hunk 2 above)
Comment (JamieMagee) [reply pkgs/development/python-modules/python-tado/default.nix]: Thanks!

**Pull request merged**